src/timeMachine/components/TagSelector.tsx

```diff
@@ -275,14 +275,21 @@ class TagSelector extends PureComponent<Props> {
 
 const mstp = (state: AppState, ownProps: OwnProps) => {
   const activeQueryBuilder = getActiveTimeMachine(state).queryBuilder
-
-  const {
-    keys,
-    keysSearchTerm,
-    keysStatus,
-    valuesSearchTerm,
-    valuesStatus,
-  } = activeQueryBuilder.tags[ownProps.index]
+  let keys = []
```

Comment:
what was all this for?

Reply:
This was because the view that we generate doesn’t map out all the properties that are implied within the reducer so a cell crashes when switching from the script editor to the query builder in the VEO. These guards prevent that